### PR TITLE
Added Croppa::render() plus comments typos

### PR DESCRIPTION
**Pull request by dannoso**
I took the time to implement the #111 
Please review and merge.

### CODE REVIEW

**Comment by dannoso** [comment]
@weotch any chance to have this merged?

File: src/Bkwld/Croppa/Handler.php [hunk 1]
```diff
@@ -104,6 +104,50 @@ public function handle($request) {
...
 	}
 
+
+	/**
+	 * Render image directly
+	 *
+	 * @param string $imagepath The imagepath!
+	 * @return boolean
+	 */
+	public function render($imagepath) {
```
Comment (weotch) [inline src/Bkwld/Croppa/Handler.php]: I think it makes sense for the new logic to exist in `Handler`, like you have it here, but I don't like how this duplicates so much of the code from `handle()`.  I think the code that `render()` shares with `handle()` should be DRYed up to it's own method, like `renderFromPath($path)` or something.

File: src/Bkwld/Croppa/Helpers.php [hunk 2]
```diff
@@ -68,17 +68,33 @@ public function tag($url, $width = null, $height = null, $options = null) {
...
+	 * @param string $url URL of an image that should be cropped
+	 * @param integer $width Target width
+	 * @param integer $height Target height
+	 * @param array $options Additional Croppa options, passed as key/value pairs.  Like array('resize')
+	 * @return string The new path to your thumbnail
+	 * @see Bkwld\Croppa\URL::generate()
+	 */
+	public function render($croppaurl) {
+		global $app;
+		return $app['Bkwld\Croppa\Handler']->render($croppaurl);
```
Comment (weotch) [inline src/Bkwld/Croppa/Helpers.php]: I've been trying to use dependency injection for all classes and referring to the app binding here breaks that.  Instead, I think `Handler` should be injected into the constructor and used here.  Sucks to have another dependency, but /shrug

**Comment by weotch** [comment]
Hey, sorry, somehow I missed the email notification of your PR!  I made some comments on the commits on stuff I'd want to see tweaked.  Thanks!

File: src/Bkwld/Croppa/Handler.php (hunk 1 above)
Comment (dannoso) [inline src/Bkwld/Croppa/Handler.php]: Yeah I see. It was the dirty quick solution. I'll refactor it.

**Comment by dannoso** [comment]
@weotch done! Have a look, please.

**Comment by weotch** [comment]
Continuing work in #125.  Thanks!

**Pull request closed without merging**